### DISCUSSION TITLE
Problems with the Hires Fix

### DISCUSSION THREAD

**Discussion by BlessedDuelOP** [TheLastBen/fast-stable-diffusion#2630] [Q&A]
Hello guys, I wanted to ask for a bit of help with this issue if possible. I'm using Collab Pro to generate my images, and I'm experiencing errors when using HiresFix. It seems these problems are associated with the amount of memory allowed by Collab. This didn't happen to me before, and I think it has to do with Xformers. Is there a way to include it in this Google Collab notebook? I would really appreciate your help with a code cell for that installation.

By the way: I'm using the Google Collab T4 GPU outwith google drive, and on another note, I want to express my gratitude to TheLastBen, who continues to update the Google Collab notebooks even though they are now paid

**Comment by TheLastBen** [comment]
use the latest notebook and check if the issue persists

**Reply by BlessedDuelOP** [reply]
I've been testing, and the 'hires fix' works with resolutions of 512x512 and 512x768. However, when I switch to 768x1024, which is what I used before, it doesn't work. I still think that maybe installing Xformers could fix this. Thanks a lot anyway for the help, it's greatly appreciated <3

**Reply by TheLastBen** [reply]
make sure xformers is being used, it is activated in the latest commit

**Reply by BlessedDuelOP** [reply]
Friend, thank you very much for putting the xformers in your google Collab; now I can do hiresfix at the resolution I like. Many, many thanks; I appreciate it a lot.
